docs/docs/responses.mdx

```diff
@@ -102,6 +101,26 @@ responses:
 In this example, when `utter_greet` gets predicted as the next action, Rasa will randomly pick one 
 of the two response variations to use.
 
+#### IDs for Response Variations
```

Comment:
Sorry to be a pain, but the ids can be used for any response, not only CRVs, so we need a section that is generalizable, so I'd make the below change and move it up after defining responses section.
```suggestion
### IDs for Responses
```